magento/magento2
# Fix issue 19647 - Backend Marketing Store View Specific Labels Options Label not aligned

### Description (*)

Backend Marketing Store View Specific Labels Options Label not aligned
### Fixed Issues (if relevant)

1.magento/magento2#19647: Backend Marketing Store View Specific Labels Options Label not aligned

### Manual testing scenarios (*)

1.Open Backend
2.Marketing -> Cart Price Rules -> Labels -> Store View Specific Labels
3.Check Options Alignment

### Contribution checklist (*)
